db/zkdbSchema.sql

```diff
@@ -29,6 +29,8 @@
 -- Server version: 10.0.29-MariaDB-cll-lve
 -- PHP Version: 5.6.36
 
+use zkdb;
```

Comment:
Remove "use zkdb".  This script can be used to setup the schema in any database, not just zkdb.  For example, we use it in prod to setup stage and training as well.  The user specifies the target database when he runs the script.